client/my-sites/checkout/checkout/redirect-payment-box.jsx

```diff
@@ -215,6 +239,35 @@ export class RedirectPaymentBox extends PureComponent {
 		} );
 	}
 
+	renderWechatScanText() {
+		if ( cartValues.cartItems.hasRenewalItem( this.props.cart ) ) {
+			return translate( 'Scan the barcode to confirm your %(price)s subscription with %(paymentProvider)s.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

Reply:
Here too - a comment (though I'm not sure it's needed at all - seems pretty self-explanatory to me).